### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by ceskyDJ**
Hi,

I updated Czech translation and added Czech version of Google play descriptions. I don't have screenshots from Czech UI, so I added English ones. Change it if you are not OK with this, please. After releasing a new version with Czech lang, I could add screenshots but I'm too lazy for compiling it and installing on my phone manually. I'm sorry for that.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -29,7 +29,8 @@
         Paulo Schopf\n
         Joaquim Pedroto Costa\n
         @madcrow\n
-        @Tubix
+        @Tubix\n
+        ceskyDJ
```
Comment (ogarcia) [inline app/src/main/res/values/strings.xml]: Here if you add your GitHub name, please add at sign. You can put _@ceskyDJ_ or, if prefer without at, your name _Michal Šmahel_.

**Review by ogarcia — CHANGES_REQUESTED** [review]
No problem with screenshots. Take note that these info is not for Google Play, it is for F-Droid. But I need that you fix your username in contributors.

**Comment by ceskyDJ** [comment]
Ok, thanks for review. I'll change the username to your preferred form.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (ceskyDJ) [reply app/src/main/res/values/strings.xml]: Ok, I changed it in commit ae3cd7bc.

**Review by ogarcia — APPROVED** [review]

**Pull request merged**